Integrations/LogRhythmRest/LogRhythmRest.yml

```diff
@@ -657,20 +657,20 @@ script:
       required: false
       secret: false
     deprecated: false
-    description: Retrieves a list of events by alarm id
+    description: Returns a list of events, by alarm ID.
     execution: false
     name: lr-get-alarm-events
     outputs:
     - contextPath: Logrhythm.Alarm.Event
-      description: Alarm event information
+      description: Alarm event information.
       type: String
     - contextPath: Logrhythm.Alarm.ID
-      description: The alarm ID
+      description: The alarm ID.
       type: String
   isfetch: false
   runonce: false
   script: '-'
   type: python
 tests:
 - LogRhythm REST test
-releaseNotes: add lr-get-alarm-data and lr-get-alarm-events commands
+releaseNotes: Added the lr-get-hosts, lr-get-alarm-data, lr-get-alarm-events, lr-get-networks, and lr-get-persons commands.
```

Comment:
@adi88d 
pull from master, move the releaseNotes to a file and remove it from here.